chapters/annotations.tex

```diff
@@ -546,21 +571,23 @@ \subsection{Single Use of Class}\label{annotation-for-single-use-of-class}\label
 For state machines it is useful to have single instances of local classes.
 This can be done using:
```

Comment:
Replacement for detail removed below:
```suggestion
To this end a classy may use the annotation:
```

Reply:
A bit of spelling error there, will add with out it.